modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -50,16 +48,24 @@ spec:
       version: v1
       resource: deployments
 ----
-
-A service binding request is created and the {servicebinding-title} controller projects the database service connection information into the application deployment as files by using a volume mount. After the request is successful, the application is redeployed and the connection is established.
-
++
+A service binding request is created. When the database service connection request succeeds, the application is redeployed and the connection is established.
```

Comment:
Modify this line as: 
A service binding request is created and a binding connection is created through a `ServiceBinding` resource. When the database service connection request succeeds, the application is redeployed and the connection is established.